### PR TITLE
files added

### PR DESCRIPTION
**Pull request by OSAMA478**

### CODE REVIEW

**Review by mujeeb100 — APPROVED** [review]
lgtm

**Review by syedabdullahali — COMMENTED** [review]
nice

**Review by syedabdullahali — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+		<meta name="viewport" content="width", initial-scale=1.0">
+		<link rel="stylesheet" href="style.css" />
+		<title>border animation</title>
+	</head>
+	<body>
+		<div class="frame">
+			<div class="rotate"></div>
+			<div class="cant">
+				<div class="area">
+					<img src="tiktok.png" alt="" />
```
Comment (nehalskdev) [inline index.html:15]: add alt do not leave it blank

File: style.css [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+}
+
+body {
+	min-height: 100vh;
+
+	background-color: #000000;
+
+	display: flex;
+	justify-content: center;
+	align-items: center;
```
Comment (nehalskdev) [inline style.css]:
```suggestion
	display: grid;
	place-items: center;
```
you can do this task in 2 line

File: style.css [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+	position: relative;
+
+	overflow: hidden;
+
+	/* background: #ffffff; */
```
Comment (nehalskdev) [inline style.css:28]: why comment?

**Review by nehalskdev — CHANGES_REQUESTED** [review]
check comments.

**Pull request merged**